core/util/wikileaks.py

```diff
@@ -38,26 +38,27 @@ def __init__(self, q, limit=1):
 		self._links_with_data = []
 
 	def run_crawl(self):
-		page = 1
```

Comment:
Should I replace page with max_attempts everywhere? also the max_attempts need to start with 1

Reply:
actually, the page was the user-supplied parameter, which signifies the number of pages the user wants to search, I am adding it again.

Reply:
the user-supplied parameter: self.limit?

Reply:
yes, but I have handled it https://github.com/saeeddhqan/Maryam/pull/166/commits/fb9a2dd47edc503292221297f75c34bc08c78391